pywinauto/controls/uia_controls.py

```diff
@@ -1017,3 +986,216 @@ def check_button(self, button_identifier, make_checked, exact=True):
+
+
+# ====================================================================
+class TreeViewWrapper(uiawrapper.UIAWrapper):
+
+    """Wrap an UIA-compatible Tree control"""
+
+    _control_types = [
+        IUIA().UIA_dll.UIA_TreeControlTypeId
+    ]
+
+    # -----------------------------------------------------------
+    def __init__(self, elem):
+        """Initialize the control"""
+        super(TreeViewWrapper, self).__init__(elem)
+
+    @property
+    def writable_props(self):
+        """Extend default properties list."""
+        props = super(TreeViewWrapper, self).writable_props
+        props.extend(['item_count'])
+        return props
+
+    # -----------------------------------------------------------
+    def item_count(self):
+        """Return a number of items in TreeView"""
+        return len(self.descendants(control_type="TreeItem"))
+
+    # -----------------------------------------------------------
+    def roots(self):
+        """Return root elements of TeeView"""
+        return self.children(control_type="TreeItem")
+
+    # -----------------------------------------------------------
+    def get_item(self, path, exact=False):
+        """Read a TreeView item
+
+        * **path** a path to the item to return. This can be one of
+          the following:
+
+          * A string separated by \\ characters. The first character must
+            be \\. This string is split on the \\ characters and each of
+            these is used to find the specific child at each level. The
+            \\ represents the root item - so you don't need to specify the
+            root itself.
+          * A list/tuple of strings - The first item should be the root
+            element.
+          * A list/tuple of integers - The first item the index which root
+            to select. Indexing always starts from zero: get_item((0, 2, 3))
+
+        * **exact** a flag to request exact match of strings in the path
+          or apply a fuzzy logic of best_match thus allowing non-exact
+          path specifiers
+        """
+
+        if not self.item_count():
+            return None
+
+        # Ensure the path is absolute
+        if isinstance(path, six.string_types):
+            if not path.startswith("\\"):
+                raise RuntimeError(
+                    "Only absolute paths allowed - "
+                    "please start the path with \\")
+            path = path.split("\\")[1:]
+
+        current_elem = None
+
+        # find the correct root elem
+        if isinstance(path[0], int):
+            current_elem = self.roots()[path[0]]
+        else:
+            roots = self.roots()
+            texts = [r.window_text() for r in roots]
+            if exact:
+                if path[0] in texts:
+                    current_elem = roots[texts.index(path[0])]
+                else:
+                    raise IndexError("There is no root element equal to '%s'" % path[0])
```

Comment:
`.format()` method is preferable for string interpolation (QC warning). Here and below.